libs/utils/utils.py

```diff
@@ -12,3 +14,8 @@ def validate_file_exists_and_is_not_empty(file_path):
 
 def is_list_recursively_empty(l):
     return all(is_list_recursively_empty(i) if isinstance(i, list) else False for i in l)
+
+def config_to_boolean(value):
+    if value.lower() not in BOOLEAN_STATES:
+        raise ValueError("Invalid boolean value")
+    return BOOLEAN_STATES[value.lower()]
```

Comment:
Can we add the newline? (Otherwise, my local linter will warning me :joy: )

Reply:
```suggestion
    return BOOLEAN_STATES[value.lower()]

```